src/renderer/App.vue

```diff
@@ -7,10 +7,17 @@
       rightAligned: isRightAligned
     }"
   >
-    <top-nav ref="topNav" />
-    <side-nav ref="sideNav" />
+    <top-nav
+      ref="topNav"
+      role="navigation"
+    />
+    <side-nav
+      ref="sideNav"
+      role="navigation"
+    />
```

Comment:
It probably makes more sense to place the roles, inside the actual components, instead of relying on Vue's fallthrough attributes behaviour.